app/src/main/res/layout/fragment_profile.xml

```diff
@@ -51,20 +51,33 @@
     <Switch
         android:id="@+id/switchl"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="50dp"
```

Comment:
you can start moving dimensions to a dimension file
https://developer.android.com/guide/topics/resources/more-resources#Dimension